tensorflow/lite/micro/kernels/svdf_test.cc

```diff
@@ -912,7 +924,7 @@ TF_LITE_MICRO_TEST(SvdfQuantized1x16Input64x1OutputReluShouldMatchGolden) {
       input_sequences_quantized,
       sizeof(tflite::testing::input_data_16x1x1) / sizeof(float),
       tflite::testing::golden_output_16x1x1, golden_quantized,
```

Comment:
We should be using tflite::testing::golden_output_relu_16x1x1 as our golden, not golden_output_16x1x1.

This fell through the cracks because the quantization scheme I used automatically applies a relu by selecting a zero point of -128, effectively applying the relu to the golden.

We should un-ifdef this golden and use it in this test.

Reply:
This test case was already being run with xtensa because the it passes with either `golden_output_relu_16x1x1` or `golden_output_16x1x1` which makes me nervous. 

changed to use `golden_output_relu_16x1x1`.